src/EnergyPlus/HVACVariableRefrigerantFlow.cc

```diff
@@ -5958,29 +5958,41 @@ namespace HVACVariableRefrigerantFlow {
 			return;
 		}
 
-		// Get result when DX coil is off
-		PartLoadRatio = 0.0;
+		// Get result when DX coil is operating at the minimum PLR (1E-20) if not otherwise specified
+		PartLoadRatio = VRFTU( VRFTUNum ).MinOperatingPLR;
```

Comment:
OK, so what was happening. This user specified 0 no load flow rate in the TU object. The model first checks PLR = 0 then PLR = 1. For a 0 no load air flow input, the output at PLR=0 is 0 watts, but the output at PLR = 1E-20 is much greater than 0 (model discontinuity). The model iterates to find the PLR to meet the load. When heating loads are small, at the operating heating air flow rate (PLR>0), the fan heat can exceed the load. Remember that the VRF TU operates the entire time step so at full load flow there will be fan heat easily greater than a few watts. The fix was to use a minimum PLR, which has been asked for by manufacturers. I set the default to 1.0E-20 until an input is provided so users can select their own value.

QUESTION: should I add the input for min PLR now? Anytime the output at min PLR exceeds the load the TU will be off and operate at the no load air flow rate.